Mathlib/Algebra/BigOperators/Fin.lean

```diff
@@ -181,6 +171,74 @@ theorem prod_trunc {M : Type*} [CommMonoid M] {a b : ℕ} (f : Fin (a + b) → M
   rw [prod_univ_add, Fintype.prod_eq_one _ hf, mul_one]
   rfl
 
+section Interval
+
+variable {M : Type*} [CommMonoid M] {n : ℕ} {v : Fin (n + 1) → M}
+
+@[to_additive]
+theorem prod_Ioi_zero : ∏ i ∈ Ioi 0, v i = ∏ j : Fin n, v j.succ := by
+  rw [Ioi_zero_eq_map, Finset.prod_map, val_succEmb]
+
+@[to_additive]
+theorem prod_Ioi_succ (i : Fin n) : ∏ j ∈ Ioi i.succ, v j = ∏ j ∈ Ioi i, v j.succ := by
+  rw [Ioi_succ, Finset.prod_map, val_succEmb]
+
+@[to_additive (attr := simp)]
+theorem prod_Ici_zero : ∏ i ∈ Ici 0, v i = ∏ j : Fin (n + 1), v j := by
+  rw [← bot_eq_zero, Ici_bot]
```

Comment:
We don't really want this lemma as stated because it has nothing to do with `∏`. It is merely an equality of the index sets of two `∏`. Instead of this lemma, you should add `Ici 0 = univ` as a simp lemma